src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -52,7 +52,7 @@ by Ben Inglis:
 | TotalReadoutTime               | RECOMMENDED. This is actually the "effective" total readout time , defined as the readout duration, specified in seconds, that would have generated data with the given level of distortion. It is NOT the actual, physical duration of the readout train. If `EffectiveEchoSpacing` has been properly computed, it is just `EffectiveEchoSpacing * (ReconMatrixPE - 1)`.<sup>3</sup> . This parameter is REQUIRED if corresponding "field/distortion" maps acquired with opposing phase encoding directions are present (see 8.9.4).                                                                                                                                                                                                                                                                                                  |
 
 <sup>2</sup>Conveniently, for Siemens’ data, this value is easily obtained as
-1/\[`BWPPPE` \* `ReconMatrixPE`\], where BWPPPE is the
+1/\[`BWPPPE` \* `ReconMatrixPE`], where BWPPPE is the
```

Comment:
Does the `]` still render correctly after removing the escape character? If yes, why not remove the other escape character right before `[`BWPPPE`` as well?

Reply:
I don't remember making this change, it was accidental, I think. Renders fine with or without escape characters but have reverted it back to include escape character to be consistent. Thanks for pointing this out.